ffi/src/sspi/utils.rs

```diff
@@ -30,10 +30,22 @@ pub fn raw_wide_str_trim_nulls(raw_str: &mut Vec<u8>) {
 }
 
 pub fn hostname() -> Result<String> {
-    whoami::fallible::hostname().map_err(|err| {
-        Error::new(
-            ErrorKind::InternalError,
-            format!("can not query the system hostname: {:?}", err),
-        )
-    })
+    // We run tests with Miri. Miri is the Rust's mid-level intermediate representation interpreter.
+    // It is unable to execute system calls. Thus, Miri cannot execute `whoami::fallible::hostname()`.
+    // So, we decided to keep hardcoded hostname value for tests.
+    #[cfg(test)]
+    {
+        Ok("test-vm".into())
+    }
+    #[cfg(not(test))]
```

Comment:
suggestion: You could use the `miri` cfg instead, so we don’t stub this code when miri is not used
```suggestion
    #[cfg(miri)]
    {
        Ok("test-vm".into())
    }
    #[cfg(not(miri))]
```

Reply:
Wow, didn't know about it. Got it!

Reply:
done